dsf-bpe/dsf-bpe-server/src/main/java/org/highmed/dsf/bpe/spring/config/MasterPatientIndexConfig.java

```diff
@@ -0,0 +1,31 @@
+package org.highmed.dsf.bpe.spring.config;
+
+import java.util.NoSuchElementException;
+
+import org.highmed.mpi.client.MasterPatientIndexClientFactory;
+import org.highmed.mpi.client.MasterPatientIndexClientServiceLoader;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class MasterPatientIndexConfig
+{
+	@Value("${org.highmed.dsf.bpe.mpi.webservice.factory.class}")
```

Comment:
Add stub implementation as default value
```suggestion
	@Value("${org.highmed.dsf.bpe.mpi.webservice.factory.class:org.highmed.mpi.client.stub.MasterPatientIndexClientStub }")
```